frontend/src/metabase/public/containers/PublicOrEmbeddedQuestion/PublicOrEmbeddedQuestionView.stories.tsx

```diff
@@ -0,0 +1,142 @@
+// @ts-expect-error There is no type definition
```

Comment:
I based the file off the dashboard version and delete what's not used, and add some necessary props.